backend/core/types/src/backend-swagger.ts

```diff
@@ -1625,7 +1649,7 @@ export class MultiselectQuestionsService {
     })
   }
   /**
-   * Get Listings by multiselect question id
+   * Get multiselect question by id
```

Comment:
Doesn't this return listings (Promise<Listing[]>) rather than multiselect questions? The previous comment looked good to me

Reply:
This file is generated code and its origins didn't change https://github.com/bloom-housing/bloom/pull/3172/files#diff-567f50929d289f46dcd4ddbc4cefa49608cb271fedff7638f38c63ab2ec5b096L1614 so hmm.

Oh looked into it more just now,  it does look like @ludtkemorgan manually updated this file in https://github.com/bloom-housing/bloom/commit/1d375355db1f94565e813ed1853b744acec5a6da 2 weeks ago. Not super high stakes, but IMO I think we should keep the gencode version and favor updating the underlying multiselect question controller